.gitignore

```diff
@@ -30,4 +30,4 @@ migrate_working_dir/
 build/
 
 # FVM
-.fvm/flutter_sdk
+.fvm/
```

Comment:
You probably need to delete `.fvm/fvm_config.dart` from repository.

Reply:
Yes, you are right. Fix this